### ISSUE TITLE
Search applications always show tag

### ISSUE THREAD

**Issue by Metroid-Prime** [adirh3/Fluent-Search#1]
(Confirmed as bug by @nomad-0 on Discord)  Turning off search applications in settings does not prevent them from showing up as a tag when searched.  For example, I don't use To Do.  I turned off enable application in settings, turned off search "search tag only", but FS still triggers "Search to do" when I type in "todo+ and hit space.  This is  reproducible for all the search applications.

EDIT: I can confirm, but this happens to ALL of the  search apps and plugins, not just translate! 
This is a high priority bug
![image](https://user-images.githubusercontent.com/1836362/120915393-3b585b00-c658-11eb-84b5-3024a0af9400.png)

**Comment by mochigithub** [comment]
I'll close mine. I've linked mine to issue #38

**Comment by mochigithub** [comment]
I can confirm, but this happens to ALL of the  search apps and plugins, not just translate! 
This is a high priority bug
![image](https://user-images.githubusercontent.com/1836362/120915393-3b585b00-c658-11eb-84b5-3024a0af9400.png)

**Comment by mochigithub** [comment]
Reopening issue per Najah's/MrF's request as this ticket is more detailed

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 0.9.85.20

**Comment by Metroid-Prime** [comment]
> This issue has been potentially fixed in version 0.9.85.20

Nomad asked me to check this.  @adirh3  I cannot confirm fix unfortunately. 

latest version:
![image](https://user-images.githubusercontent.com/1836362/120960525-df9dd880-c710-11eb-9c06-81ba93bc678a.png)


Setting is turned off.
![image](https://user-images.githubusercontent.com/1836362/120960453-b8470b80-c710-11eb-9e0d-df6b57ec6ba3.png)

ToDo still firing: 
![image](https://user-images.githubusercontent.com/1836362/120960504-d6ad0700-c710-11eb-91ff-84c30fe3d790.png)


Happening to all others too, including UI:
![image](https://user-images.githubusercontent.com/1836362/120960492-cdbc3580-c710-11eb-8246-b31c22158475.png)

**Comment by Nomad-0** [comment]
I'm having weird issues regarding this
Seeing _to do_ tag whether it's enabled or disabled
-disabled
![image](https://user-images.githubusercontent.com/76395594/120960980-bb78e080-c77e-11eb-9fdd-92c1801e4ff6.png)
![image](https://user-images.githubusercontent.com/76395594/120960902-96846d80-c77e-11eb-9a19-8962a116e9a7.png)

-enabled
![image](https://user-images.githubusercontent.com/76395594/120961061-e19e8080-c77e-11eb-96b5-388f7a14d506.png)
![image](https://user-images.githubusercontent.com/76395594/120961035-d9dedc00-c77e-11eb-9cbf-83100ca2a184.png)

but I'm **not seeing** or being able to use _UI_ tag whether it's enabled or disabled
-enabled
![image](https://user-images.githubusercontent.com/76395594/120961177-1a3e5a00-c77f-11eb-822f-83aabdfa556a.png)
![image](https://user-images.githubusercontent.com/76395594/120961198-24605880-c77f-11eb-8aec-9fb61a455d7d.png)

**Comment by Nomad-0** [comment]
**update**
suddenly I'm not seeing _to do_ anymore, whether it's enabled or disabled
-enabled
![image](https://user-images.githubusercontent.com/76395594/120961347-815c0e80-c77f-11eb-841a-ab420664955c.png)
![image](https://user-images.githubusercontent.com/76395594/120961365-891bb300-c77f-11eb-9dd6-96c60c8aad05.png)

-disabled
![image](https://user-images.githubusercontent.com/76395594/120961396-95077500-c77f-11eb-85e3-3be7695ae6b4.png)
![image](https://user-images.githubusercontent.com/76395594/120961413-9cc71980-c77f-11eb-9060-2621b1024394.png)

**Comment by adirh3** [comment]
This_issue_has_been_potentially_fixed_in_version_

**Comment by Nomad-0** [comment]
issue fixed on my end. @Metroid-Prime please check.

**Comment by Metroid-Prime** [comment]
@adirh3 @Nomad-0  
Unfortunately I am still having this issue in v0.9.85.25 
![image](https://user-images.githubusercontent.com/1836362/121296402-76030300-c8a5-11eb-9c12-c7fcdd10b50a.png)

**Comment by Nomad-0** [comment]
> @adirh3 @Nomad-0
> Unfortunately I am still having this issue in v0.9.85.25

do you mean version 0.9.85.35?

**Comment by Metroid-Prime** [comment]
> > @adirh3 @Nomad-0
> > Unfortunately I am still having this issue in v0.9.85.25
> 
> do you mean version 0.9.85.35?

Yes, thank you @Nomad. Corrected my original post.

**Comment by adirh3** [comment]
@Nomad-0 That's a tag for a folder called todo, the issue with the search application is fixed.
@Metroid-Prime it applies only after restart.

**Comment by Metroid-Prime** [comment]
@adirh3  How about this one?
![image](https://user-images.githubusercontent.com/1836362/121308861-604a0980-c8b6-11eb-82f3-20b566e2966c.png)

![image](https://user-images.githubusercontent.com/1836362/121308878-663fea80-c8b6-11eb-96c8-d6c140588028.png)

**Comment by adirh3** [comment]
> @adirh3 How about this one?
> ![image](https://user-images.githubusercontent.com/1836362/121308861-604a0980-c8b6-11eb-82f3-20b566e2966c.png)
> 
> ![image](https://user-images.githubusercontent.com/1836362/121308878-663fea80-c8b6-11eb-96c8-d6c140588028.png)

The "settings" search tag is being used to search for FS settings, it will not be removed.
You sure "Enabled application" was disabled when searching with that tag? if it did, then open an issue for that, since you shouldn't get Windows settings results if you disabled the app

**Comment by adirh3** [comment]
@Metroid-Prime Can we close this?

**Comment by Metroid-Prime** [comment]
it's still happening on virtually every search

I'm able to replicate it on every search.
![image](https://user-images.githubusercontent.com/1836362/121787800-3118ee00-cb7d-11eb-9c44-b6c476430cd6.png)


https://media.discordapp.net/attachments/851107458339700736/853360882381357086/unknown.png

**Comment by Metroid-Prime** [comment]
> The "settings" search tag is being used to search for FS settings, it will not be removed.

"settings" search tag is searching my SYSTEM settings, which again is turned off in my preferences!  What am I doing wrong here?
![image](https://user-images.githubusercontent.com/1836362/121787894-bb615200-cb7d-11eb-857e-d1afac74176b.png)


The settings show everything is turned off so "settings" + space should not be triggering anything

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 0.9.85.45

**Comment by Metroid-Prime** [comment]
The new version still has the problem, but only with the TAB button. Space no longer exhibits the issue

**Comment by adirh3** [comment]
@Metroid-Prime You sure you're not confusing with FS settings? 
@Nomad-0 Can you reproduce it?

**Comment by Nomad-0** [comment]
> @Nomad-0 Can you reproduce it?

No, I can't

**Comment by Metroid-Prime** [comment]
I believe this issue has now 'shifted' in the latest version if that makes any sense.

When the search tags are turned off, Fluent Search is searching for the "next tag that's on".  For example, when I typed in "todo" and hit "tab", it triggers the command tab.
![image](https://user-images.githubusercontent.com/1836362/121955415-ca2e3d00-cd14-11eb-877a-0e768cf90a63.png)


Then I went to the Command setting and turned enable application OFF, triggered FS and typed "todo" + tab again  and now it triggers the App search tag which I have enabled.
![image](https://user-images.githubusercontent.com/1836362/121955508-e7630b80-cd14-11eb-8851-5e73f7e02010.png)
 
 When a search tag is turned off, for example todo, typing "todo" + hitting TAB should not trigger anything. This is confusing interface
